### PR TITLE
CIGI-208: Add feature medium component

### PR DESCRIPTION
**Pull request by johnbxu**
#### Description of changes
resolves CIGIHub/cigi-tickets#208

image

image

- Create styling and template for featured medium component
- Added styling for multimedia button on large and medium featured components
- Use /opinions to test

### CODE REVIEW

**Comment by gabrieltanchen** [comment]
@johnbxu failing tests

**Comment by johnbxu** [comment]
Fixed migration as per instructions on https://docs.wagtail.io/en/stable/releases/2.11.html

File: articles/models.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ class ArticleLandingPageFeaturedArticle(Orderable):
         related_name='featured_articles',
     )
     article_page = models.ForeignKey(
-        'articles.ArticlePage',
+        'wagtailcore.Page',
```
Comment (gabrieltanchen) [inline articles/models.py:61]: Why is this changed to the general page?

**Review by gabrieltanchen — CHANGES_REQUESTED** [review]

File: articles/models.py (hunk 1 above)
Comment (johnbxu) [reply articles/models.py:61]: It's for testing different featurable content on the opinion landing page, ex. multimedia.

Comment (johnbxu) [reply articles/models.py]: I changed this back to ArticlePage and added a featured_content field to HomePage to test out featured content. This broke some tests that needed a homepage fixture though. I'll look into fixing it after finishing the drupal promotion blocks.

**Comment by gabrieltanchen** [comment]
@johnbxu don't worry about adding featured content to the HomePage. I'm already working on that. Just reverting the ArticleListPage featured content foreign key to ArticlePage is good enough.

**Comment by johnbxu** [comment]
Removed home page feature and reverted featured_article model back to ArticlePage.

**Review by gabrieltanchen — APPROVED** [review]

**Pull request merged**